src/main/java/core/basesyntax/service/impl/ToCsvFileReportWriterImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.service.ToCsvFileReportWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+public class ToCsvFileReportWriterImpl implements ToCsvFileReportWriter {
+
+    @Override
+    public void writeReport(File report, FruitsDao fruitsDao, String columnsNamesLine) {
```

Comment:
```suggestion
    public void writeReport(String report, String header) {
```

Let this method be responsible just for writing data to file